mrblib/mitamae/resource_executor/base.rb

```diff
@@ -163,12 +163,17 @@ def check_command(*args)
       end
 
       def run_specinfra(type, *args)
```

Comment:
There might be a case where the backend can run `check_foo_is_bar(arg1)` but cannot run `check_foo_is_bar(arg2)` .
That's why I decided to pass all arguments to `find_inline_backend` and `runnable?`.
But I cannot provide a real-world example for now.